src/services/tasks/types.ts

```diff
@@ -39,6 +39,37 @@ export type TaskInput = Pick<
   'functionName' | 'data' | 'startTimestamp' | 'priority' | 'tags'
 >;
 
+export enum ApiFunctionRequestMethod {
+  'GET' = 'GET',
+  'POST' = 'POST',
+  'PUT' = 'PUT',
+  'DELETE' = 'DELETE',
+  'PATCH' = 'PATCH',
+  'OPTIONS' = 'OPTIONS',
+  'HEAD' = 'HEAD',
+}
+
+export interface ApiFunctionRequestObject {
+  version: '2.0';
+  rawPath: string;
+  rawQueryString: string;
+  headers: Record<string, string>;
+  requestContext: {
+    http: {
+      method: ApiFunctionRequestMethod;
+    };
+  };
+  body: string;
+  isBase64Encoded: boolean;
+}
+
+export interface ApiFunctionResponseObject {
+  statusCode: number;
+  headers?: Record<string, string>;
+  body?: string;
+  isBase64Encoded?: boolean;
+}
```

Comment:
Like we did for the Auth Service during the OpenID Connect epic, can these properties be described with JSDoc? For example: https://github.com/ExtraHorizon/javascript-sdk/blob/dev/src/services/auth/applications/types.ts#L172-L189
We argued back then that the documentation within the SDK might even be more valuable than the public documentation, especially with regard to more detailed information like the fields.